src/Microsoft.Fast.Components.FluentUI/Components/NavMenu/FluentNavMenu.razor

```diff
@@ -2,8 +2,8 @@
 
 @inherits FluentComponentBase
 <div>
-    <CascadingValue Value="this" Name="NavMenu" IsFixed="true">
-        <CascadingValue Value="Expanded" Name="NavMenuExpanded">
+    <CascadingValue Value="this" IsFixed="true">
+        <CascadingValue Value="@Expanded" Name="NavMenuExpanded">
```

Comment:
Made sure it is the value of the `Expanded` member, in case someone renames it in future it will refactor instead of passing a string `"Expanded"`.